### DISCUSSION TITLE
3.2.0 Beta 3

### DISCUSSION THREAD

**Discussion by sirjuddington** [sirjuddington/SLADE#1257] [Releases]
**General**
- Updated ZDoom language definitions (thanks @Blue-Shadow)
- Updated Eternity EDF definitions (thanks @Altazimuth)
- Fixed lua print override (thanks @flarn2006)

**Resource Editor**
- Added the ability to convert to PSX DOOM format textures (thanks @BodbDearg)
- Fixed a crash when creating an entry/dir within an empty dir on linux
- Fixed a crash on updating the entry list filter on linux
- Correctly set entry as modified if offsets were changed in the gfx editor
- Fixed directory archives not being marked as saved after saving
- Fixed renaming entries with / or \ character in wad archives
- Fixed being unable to undo/redo entry modifications if changes were saved via the "Save Changes?" prompt
- Fixed undo/redo entry creation ignoring the extension

**Texture Editor**
- Fixed double closing brackets when saving TEXTURES
- Set the default texture type to 'Texture' instead of a blank string, also when converting from TEXTUREx
- Fixed reading TEXTURES definitions with 'optional' keyword

**Map Editor**
- Fixed std::bad_variant_access exception on certain operations (thanks @anotak)
- Various improvements and fixes to texture handling and precedence (thanks @Talon1024)
- Fixed an error when loading a custom port config from the user dir

<hr /><em>This discussion was created from the release <a href='https://github.com/sirjuddington/SLADE/releases/tag/3.2.0_b3'>3.2.0 Beta 3</a>.</em>

**Comment by flarn2006** [comment]
I was worried development had been mostly abandoned - glad to see that's not the case!
